packages/hackathon/src/models/Projects.ts

```diff
@@ -258,7 +281,7 @@ export class Project extends SheetRow<Project> {
   }
 
   async addJudge(hacker: Hacker) {
-    if (!hacker.canJudge) throw new SheetError(`${hacker.name} is not a judge`)
+    if (!hacker.canJudge) throw new Error(`${hacker.name} is not a judge`)
```

Comment:
We should probably create an `ArtifactError` class or `HackathonError` class or something and use that instead.

Reply:
Yeah a TODO for the future. It needs to be made in wholly artifact package like SheetError is in whollySheet.